packages/http-client-java/emitter/src/operation-utils.ts

```diff
@@ -200,8 +200,12 @@ export function cloneOperationParameter(parameter: Parameter): Parameter {
   );
 }
 
-function operationIs(operation: Operation, name: string | undefined, namespace: string): boolean {
-  let currentOp: Operation | undefined = operation;
+function operationIs(
+  operation: SdkHttpOperation,
+  name: string | undefined,
+  namespace: string,
+): boolean {
+  let currentOp: Operation | undefined = operation.__raw.operation;
```

Comment:
I'm also using the raw operation currently. Because `SdkHttpOperation` does not have `name` and `namespace` properties. But I checked the usages of `operationIs` function, it is only used to check if an operation is an azure core operation. So maybe we can have below helper function.
```ts
function operationIsInAzureCore(operation: SdkHttpOperation) {
  operation.crossLanguageDefinitionId.contains("Azure.Core");
}
```

Reply:
The func checks `sourceOperation` too. I guess TCGC does not have this on operation yet.

Reply:
Yes, it's similar, TCGC does not this on operation.